### DISCUSSION TITLE
Possible to add string to {{VALUE}}?

### DISCUSSION THREAD

**Discussion by ltroj** [chhoumann/quickadd#462] [Q&A]
I'm using the QuickAdd template choice to populate the following template:

```
## To do
- [ ] Triage #Topic/{{VALUE:Topic}}
```

Unfortunately this template messes up my tags pane (by adding an empty tag as well as a ``#{{VALUE:Topic}}`` tag).

Is it possible to let QuickAdd prepend the ``#Topic/`` part without having to type it out on file creation?

Something like

```
## To do
- [ ] Triage {{'#Topic/'+VALUE:Topic}}
```
I'm also open for other suggestions regarding this workflow.
Thanks!

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that.

Format tokens can't concatenate literals, but an inline script can - and it also fixes the tag-pane pollution, because the template file then contains no raw `#Topic/` at all (text inside a code fence isn't indexed as a tag). Replace the line with:

````
```js quickadd
const topic = await this.quickAddApi.inputPrompt("Topic");
this.variables["Topic"] = topic;
return `- [ ] Triage #Topic/${topic}`;
```
````

When the template expands, the fence is replaced by whatever the script returns, so the created note gets `- [ ] Triage #Topic/foo` while the template file itself stays tag-free.

The `this.variables` line is optional: it stores the answer so any `{{VALUE:Topic}}` elsewhere in the same template reuses it without prompting again.

Docs: https://quickadd.obsidian.guide/docs/InlineScripts
